pkg/start/start.go

```diff
@@ -153,16 +178,97 @@ func (o *Options) Run() error {
 	return nil
 }
 
+func (o *Options) handler() http.Handler {
```

Comment:
Can we drop this helper and just keep this inline?  I don't think saving a line in the caller is worth the indirection.